compass/ocean/tests/spherical_harmonic_transform/qu_convergence/init.py

```diff
@@ -0,0 +1,73 @@
+from compass.model import run_model
+from compass.step import Step
+
+
+class Init(Step):
+    """
+    A step for running a spherical harmonic transformation
+    for the shperical_harmonic_transfrom test case
+    """
+    def __init__(self, test_case, resolution, algorithm, order):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.spherical_harmonic_transform.qu_convergence.QuConvergence
+            The test case this step belongs to
+
+        resolution : int
+            The resolution of the (uniform) mesh in km
+
+        algorithm : str
+            Use either the 'parallel' or 'serial' algorithm
+
+        order : int
+            For algorithm = 'parallel', the order of the shperical harmonic transform
+            For algorithm = 'serial', the number of latitudes in the Gaussian grid
```

Comment:
```suggestion
            For ``algorithm = 'parallel'``, the order of the spherical harmonic
            transform.  For ``algorithm = 'serial'``, the number of latitudes in
            the Gaussian grid.
```
If you really want this to be something like a bulleted list, you would need to follow the instructions here:
https://docutils.sourceforge.io/docs/user/rst/quickref.html
Formatting these on separate lines in the docstring has no impact on how they end up in the API documentation so it's better to just assume they are a paragraph.